modules/renter/contractor/renew.go

```diff
@@ -18,6 +18,10 @@ func (c *Contractor) managedRenew(contract modules.RenterContract, numSectors ui
 	} else if host.StoragePrice.Cmp(maxStoragePrice) > 0 {
 		return modules.RenterContract{}, errTooExpensive
 	}
+	// cap host.MaxCollateral
+	if host.MaxCollateral.Cmp(maxCollateral) > 0 {
+		host.MaxCollateral = maxCollateral
+	}
```

Comment:
and this only applies to the new hosts? Not quite following how the old hosts end up not running this code.